packages/react/src/index.ts

```diff
@@ -118,10 +118,9 @@ export { default as CentralizedSliderBase } from "./molecules/centralizedSlider/
 export { default as SingleDatetimePicker } from "./molecules/datetime/single";
 export { default as DateInput } from "./molecules/datetime/dateInput";
 export { default as DateInputExample } from "./molecules/datetime/dateInputExample";
-
 export { default as HorizontalList } from "./molecules/horizontalList/base";
-
 export { default as IconList } from "./molecules/IconList/base";
+export { default as GraphBase } from "./molecules/graph/base";
```

Comment:
@ShreyDhyani Including this here doesn't export to the final package also? Making chart.js a dependency for the project?

Reply:
Removed GraphBase from Reusejs Components @rjvim

Reply:
@ShreyDhyani Does this mean it won't work in Storybook? What we are looking for here is examples in Storybook which someone can copy as snippet and start using it.

Reply:
The packages are imported in Story Book and the setup steps to use Charts are also given in docs for the graphs. After following the steps mentioned in docs, user can directly use the snippet. @rjvim